_posts/2024-06-03-passion-intentionality-c4l2024.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Passion and Intentionality: Code4Lib 2024"
+date: 2024-05-30T09:00:00
+author: Patrick Galligan
+layout: post
+categories: Conferences/Education
+tags:
+    - DevOps
+    - Manifest-no
+    - community
+
+excerpt_separator: <!--more-->
+---
+
+I went to [Code4Lib]( https://2024.code4lib.org/) again this year, and the experience reaffirmed how much I love this conference and community. As one might expect, practical applications of AI in the library and archives spaces got a lot of attention and time this year, but while these talks were fascinating, useful, and important, they didn’t resonate with me quite as much as some of the more human-oriented talks. There are so many smart people out there thinking about how to make life easier for people by technology, which I find incredibly useful and uplifting. For me, Code4Lib has become less about unique expertise, and more about impact on a larger community, as Hillel alludes to in [his recent conference report]( https://blog.rockarch.org/ai-pdfs-and-community).

```

Comment:
```suggestion
I went to [Code4Lib](https://2024.code4lib.org/) again this year, and the experience reaffirmed how much I love this conference and community. As one might expect, practical applications of AI in the library and archives spaces got a lot of attention and time this year, but while these talks were fascinating, useful, and important, they didn’t resonate with me quite as much as some of the more human-oriented talks. There are so many smart people out there thinking about how to make life easier for people by technology, which I find incredibly useful and uplifting. For me, Code4Lib has become less about unique expertise, and more about impact on a larger community, as Hillel alludes to in [his recent conference report](https://blog.rockarch.org/ai-pdfs-and-community).
```
I'm seeing a lot of leading spaces in links. I'm not sure how that impacts the link once it's rendered to HTML but you may want to look at these and just clean them up.

Reply:
Thanks! I wrote this in word and then copy/pasted. Must have been introduced there and I missed it.